musicbot/bot.py

```diff
@@ -1702,6 +1702,47 @@ def check(m):
                 delete_after=20
             )
 
+    async def cmd_remove(self, message, player, index=None):
```

Comment:
`index` should be given as a positional argument, not a kwarg. It is a required argument for the command to work, right?